ntoskrnl/kd64/kdapi.c

```diff
@@ -2199,15 +2199,15 @@ KdSystemDebugControl(
                     break;
 
             }
-            break;
+            return STATUS_SUCCESS;
         }
 #endif
 
         /* Special case for stack frame dumps */
         case 'DsoR':
         {
             KeRosDumpStackFrames((PULONG_PTR)InputBuffer, InputBufferLength);
-            break;
+            return STATUS_SUCCESS;
```

Comment:
Why those changes?

Reply:
I think this is from the merge with Timo's fork while we resolved shadow mapping, I did not make this change